### PR TITLE
Selenium 3 changes

### PR DESCRIPTION
**Pull request by RutvikChandla**

### CODE REVIEW

**Review by asambstack — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
 
+# php-selenium-browserstack
+
+## Prerequisites 
+1. php and composer should be installed in your system
```
Comment (asambstack) [inline README.md:5]:
```suggestion
1. `php` and `composer` should be installed in your system
```

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+    "realMobile" => "true",
+    "os_version" => "14.0",
+    "name" => "BStack-[Php] Sample Test", // test name
+    "build" => "BStack Build Number 1" // CI/CD job or build name
+);
+// IMP: Use your browserstack username and accesskey
+$web_driver = RemoteWebDriver::create("https://USERNAME:ACCESS_KEY@hub-cloud.browserstack.com/wd/hub",$caps);
+```
+
+3. Run tests
```
[the remark this answers is not in the document]
Comment (RutvikChandla) [reply README.md:41]: No, for parallel tests of PHP we recommend to use frameworks.

**Comment by RutvikChandla** [comment]
@asambstack Please review.

**Comment by asambstack** [comment]
Do we not have parallel tests for PHP?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (asambstack) [reply README.md:41]: >Format this point, single test and local test should be sub-headings within point-3.

This isn't done. Make single and local tests as sub headings.

[the remark this answers is not in the document]
Comment (RutvikChandla) [reply README.md:41]: @asambstack is the recent change correct?

File: README.md [hunk 3]
```diff
@@ -1 +1,63 @@
+# php-selenium-browserstack
 
+# php-selenium-browserstack
```
Comment (asambstack) [inline README.md:3]: This line can be removed.

File: README.md [hunk 4]
```diff
@@ -1 +1,63 @@
...
+	"browserVersion" => "latest",
+);
+
+// Set you credentails
+$BROWSERSTACK_USERNAME = "BROWSERSTACK_USERNAME";
+$BROWSERSTACK_ACCESS_KEY = "BROWSERSTACK_ACCESS_KEY";
+```
+
+## To run tests
+---
```
Comment (asambstack) [inline README.md:49]: This divider can be removed.

**Review by asambstack — COMMENTED** [review]
Readme changes

**Review by asambstack — APPROVED** [review]
LGTM

**Pull request merged**